pkg/app/piped/cloudprovider/cloudrun/servicemanifest.go

```diff
@@ -199,3 +201,34 @@ func parseContainerImage(image string) (name, tag string) {
 	name = paths[len(paths)-1]
 	return
 }
+
+func FindArtifactVersion(sm ServiceManifest) (*model.ArtifactVersion, error) {
```

Comment:
To be unified let's how about returning a list of version.
```suggestion
func FindArtifactVersions(sm ServiceManifest) ([]model.ArtifactVersion, error) {
```

Reply:
That sounds nice!
I did because Cloud Run can use only one image in one app.
So I'll fix to change  returning a list of version and insert comment about that.